dashboards/typescript/typescript.json

```diff
@@ -0,0 +1,694 @@
+{
```

Comment:
I'm seeing a lot of entity GUIDs and explicit time ranges in these NRQL queries. Let's get this updated to handling entities with different GUIDs and remove the explicit time ranges.